doc/concepts/data-concepts/repo.md

```diff
@@ -0,0 +1,33 @@
+# Repository
+
+A Pachyderm repository is a location where you store your data inside
+Pachyderm. A Pachyderm repository is a top-level data object that contains
+files and folders. Similar to Git, a Pachyderm repository tracks all
+changes to the data and creates a history of data modifications that you
+can access and review. You can store any type of file in a Pachyderm repo,
+including binary and plain text files.
+
+Unlike a Git repository that stores history in a `.git` file in your copy
+of a Git repo, Pachyderm stores the history of your commits in a centralized
+location in the etcd key-value store. Because of that, you do not run into
```

Comment:
```suggestion
location. Because of that, you do not run into
```

While you are correct it's stored in etcd, that info is needed here. Also, that might change actually at some point and there's no reason to need this doc to change when it does.